api.planx.uk/saveAndReturn/resumeApplication.ts

```diff
@@ -42,6 +41,11 @@ const resumeApplication = async (
   }
 };
 
+interface ValidateRequest {
+  teams: Team[];
+  sessions: LowCalSession[] | null;
```

Comment:
nit: I think typing `lowcal_sessions` as `sessions` is quite confusing given we have a new, empty, to-be-eventually-populated table `sessions` - can we be wordy but clear here and use `lowcalSessions` ?